pipewire: remove installed-test that always timeouts

pineapplehunter opened the pull request:
The installed test for pipewire has been failing for a long time.
I found that one of the test runs a infinite loop without any exit condition, which made the test always fail with a timeout.
This patch removes the test.
It should not affect any functionality of pipewire itself.

~~Additionally, this PR moves the pipewire-related files to pkgs/by-name.~~ Removed this commit because eval was failing.

closes #357004

<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [x] [NixOS tests] in [nixos/tests].
  - [x] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

K900 commented on pkgs/development/libraries/pipewire/default.nix:
```diff
@@ -241,9 +241,12 @@ stdenv.mkDerivation (finalAttrs: {
+    patchShebangs doc/*.py
+    patchShebangs doc/input-filter-h.sh
+
+    # Remove installed-test that runs forever
```
I know this is very nitpicky, but can we not batch this with unrelated changes?

pineapplehunter replied:
Would it be better to split the commit here or create a new PR?

pineapplehunter committed:
pipewire: move patchShebangs from postUnpack to postPatch

The files can be specified relative to te source root in postPatch.

pineapplehunter committed:
pipewire: remove installed-test that runs forever

Pipewire installed-test alwasy files with a timeout because the
pipewire-alsa-stress runs a infinite loop.
This patch removes the test.
